packages/teams-js/test/public/people.spec.ts

```diff
@@ -26,270 +26,233 @@ describe('peoplePicker', () => {
-      const message = mobilePlatformMock.findMessageByFunc('people.selectPeople');
-      expect(message).not.toBeNull();
-      expect(message.args.length).toBe(1);
-    });
-  });
+      it(`should allow selectPeople calls with undefined peoplePickerInputs. context: ${context}`, async () => {
+        await mobilePlatformMock.initializeWithContext(context);
+        mobilePlatformMock.setClientSupportedSDKVersion(minVersionForSelectPeople);
+        people.selectPeople((error: SdkError, people: people.PeoplePickerResult[]) => {}, undefined);
+        const message = mobilePlatformMock.findMessageByFunc('people.selectPeople');
+        expect(message).not.toBeNull();
+        expect(message.args.length).toBe(1);
+        expect(message.args[0]).toEqual(null);
+      });
 
-  it('selectPeople calls with successful result', done => {
-    mobilePlatformMock.initializeWithContext(FrameContexts.content).then(() => {
-      mobilePlatformMock.setClientSupportedSDKVersion(minVersionForSelectPeople);
-
-      people.selectPeople((e: SdkError, m: people.PeoplePickerResult[]) => {
-        expect(e).toBeFalsy();
-        expect(result.length).toBe(1);
-        const person = m[0];
-        expect(person).not.toBeNull();
-        expect(person.objectId).not.toBeNull();
-        expect(typeof person.objectId === 'string').toBeTruthy();
-        expect(typeof person.displayName === 'string').toBeTruthy();
-        expect(typeof person.email === 'string').toBeTruthy();
-        expect(person.objectId).toBe('5842943a-aa5a-470a-bfdc-7311b9988962');
-        expect(person.displayName).toBe('Sonal Jha');
-        expect(person.email).toBe('sojh@m365x347208.onmicrosoft.com');
-        done();
+      it(`selectPeople call in default version of platform support fails. context: ${context}`, done => {
+        mobilePlatformMock.initializeWithContext(context).then(() => {
+          mobilePlatformMock.setClientSupportedSDKVersion(originalDefaultPlatformVersion);
+          people.selectPeople((error: SdkError, people: people.PeoplePickerResult[]) => {
+            expect(error).not.toBeNull();
+            expect(error.errorCode).toBe(ErrorCode.OLD_PLATFORM);
+            done();
+          });
+        });
       });
 
-      const message = mobilePlatformMock.findMessageByFunc('people.selectPeople');
-      expect(message).not.toBeNull();
-      expect(message.args.length).toBe(1);
-
-      const callbackId = message.id;
-      const result = [
-        {
-          objectId: '5842943a-aa5a-470a-bfdc-7311b9988962',
-          displayName: 'Sonal Jha',
-          email: 'sojh@m365x347208.onmicrosoft.com',
-        } as people.PeoplePickerResult,
-      ];
-      mobilePlatformMock.respondToMessage({
-        data: {
-          id: callbackId,
-          args: [undefined, result],
-        },
-      } as DOMMessageEvent);
-    });
-  });
+      it(`selectPeople calls with valid peoplePickerInput. context: ${context}`, done => {
```

Comment:
Can we use async and await instead of callbacks?

Reply:
In v1, we have callbacks. I am having two groups of testing. With promises for v2, and with callbacks for v1

Reply:
Okay cool.

Reply:
IS it okay to close these comments?

Reply:
Closing this, since you have approved the PR :)